config/routes.rb

```diff
@@ -137,6 +137,9 @@
         scope module: :achievement do
           concerns :conditional
         end
+
+        get '/award' => 'achievements#award_users', on: :member
```

Comment:
Agree, I deviated to make the URL more meaningful. Otherwise, we can use this? Or do you have other suggestions?

```ruby
get '/users' => 'achievement#award_users', on: :member
patch '/users' => 'achievement#update_users', on: :member
```